### PR TITLE
[Infra] Add app/servicer keys to LocalNet genesis

### PR DESCRIPTION
**Pull request by adshmh**
### Summary generated by Reviewpad on 01 Jun 23 18:05 UTC
This pull request adds 2 applications and 2 servicers to the genesis.json file for LocalNet configuration, which can be used for testing the CLI. It is part of work on issue #754. The patch includes changes to the CHANGELOG.md and README.md files to reflect the added addresses. The relevant options for this pull request are "Documentation" and "Other," specifically "added 2 applications and 2 servicers to genesis.json for LocalNet."

## Issue

Part of work on #754 

## Type of change

Please mark the relevant option(s):

- [ ] New feature, functionality or library
- [ ] Bug fix
- [ ] Code health or cleanup
- [ ] Major breaking change
- [X] Documentation
- [x] Other Added 2 applications and 2 servicers to genesis.json for LocalNet

## List of changes
- Added 2 applications to genesis.json for LocalNet
- Added 2 servicers to genesis.json for LocalNet

## Testing

- [ ] `make develop_test`; if any code changes were made
- [ ] `make test_e2e` on [k8s LocalNet](https://github.com/pokt-network/pocket/blob/main/build/localnet/README.md); if any code changes were made
- [ ] `e2e-devnet-test` passes tests on [DevNet](https://pocketnetwork.notion.site/How-to-DevNet-ff1598f27efe44c09f34e2aa0051f0dd); if any code was changed
- [ ] [Docker Compose LocalNet](https://github.com/pokt-network/pocket/blob/main/docs/development/README.md); if any major functionality was changed or introduced
- [X] [k8s LocalNet](https://github.com/pokt-network/pocket/blob/main/build/localnet/README.md); if any infrastructure or configuration changes were made

## Required Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added, or updated, [`godoc` format comments](https://go.dev/blog/godoc) on touched members (see: [tip.golang.org/doc/comment](https://tip.golang.org/doc/comment))
- [x] I have tested my changes using the available tooling
- [ ] I have updated the corresponding CHANGELOG

### If Applicable Checklist

- [x] I have updated the corresponding README(s); local and/or global
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added, or updated, [mermaid.js](https://mermaid-js.github.io) diagrams in the corresponding README(s)
- [ ] I have added, or updated, documentation and [mermaid.js](https://mermaid-js.github.io) diagrams in `shared/docs/*` if I updated `shared/*`README(s)

### CODE REVIEW

File: build/localnet/README.md [hunk 1]
```diff
@@ -148,6 +148,21 @@ For example:
...
+#### Applications staked on LocalNet
+Applications with the following addresses are staked on LocalNet, through [applications field of the genesis.json in the configuration](https://github.com/pokt-network/pocket/blob/main/build/localnet/manifests/configs.yaml#L4088)
+
+- `00001fff518b1cdddd74c197d76ba5b5dedc0301`
+- `00101f2ff54811e84df2d767c661f57a06349b7e`
+
+These addresses can be used for e.g. testing the CLI.
+
+#### Servicers staked on LocalNet
+Servicers with the following addresses are staked on LocalNet, through [servicers field of the genesis.json in the configuration](https://github.com/pokt-network/pocket/blob/main/build/localnet/manifests/configs.yaml#L4120)
```
Comment (Olshansk) [inline build/localnet/README.md:161]:
```suggestion
Servicers with the following addresses are staked on LocalNet, through the [servicers field of the genesis.json in the LocalNet configuration](https://github.com/pokt-network/pocket/blob/main/build/localnet/manifests/configs.yaml#L4120)
```

**Review by Olshansk — APPROVED** [review]
A couple small NITs but otherwise LGTM. Once CI passes, the pattern we use is "Squash & Merge" and copy-paste the PR description into the commit message.

File: build/localnet/README.md [hunk 2]
```diff
@@ -148,6 +148,21 @@ For example:
 - `0010297b55fc9278e4be4f1bcfe52bf9bd0443f8` is a servicer #001.
 - `314019dbb7faf8390c1f0cf4976ef1215c90b7e4` is an application #314.
 
+
+#### Applications staked on LocalNet
+Applications with the following addresses are staked on LocalNet, through [applications field of the genesis.json in the configuration](https://github.com/pokt-network/pocket/blob/main/build/localnet/manifests/configs.yaml#L4088)
```
[the remark this answers is not in the document]
Comment (adshmh) [reply build/localnet/README.md]: Thank you for the review. Fixed.

File: build/localnet/README.md (hunk 1 above)
Comment (adshmh) [reply build/localnet/README.md]: Thank you for the review. Fixed.

**Comment by adshmh** [comment]
> A couple small NITs but otherwise LGTM. Once CI passes, the pattern we use is "Squash & Merge" and copy-paste the PR description into the commit message.

Thank you for the review. Updated the commit message.

**Review by Olshansk — APPROVED** [review]
@adshmh I tried to review the changes since my last review and I'm guessing you `git fooed` to clean up some of your local commits.

It really helps the reviewer to see what changes took place between each review if we don't modify the git history, especially for PRs that are larger and take longer.

Per our review guidelines (#773), we always do a "squash and merge" when we merge the PR into main, so the history will be clean on main regardless.

Screenshot 2023-05-23 at 8 52 00 PM
Screenshot 2023-05-23 at 8 51 56 PM

**Comment by Olshansk** [comment]
> > I tried to review the changes since my last review and I'm guessing you `git fooed` to clean up some of your local commits.
> 
> Sorry about that, wasn't local commits, did a push -f by mistake (the CI's timezone does not match mine, so CHANGELOG checks failed on CI after having passed locally).

No worries. Regardless, see my previous comment [here](https://github.com/pokt-network/pocket/pull/781#pullrequestreview-1440595590) about merging it in.

Seems like you might need to merge with master first since there are merge conflicts since this was last reviewed.

**Comment by Olshansk** [comment]
@adshmh Did you still want to merge this in?

**Comment by Olshansk** [comment]
@adshmh Following up on an offline discussion around merge conflicts related to changelogs.

I took the liberty of resolving the conflict (and pushing the change) and recorded a video as a reference that we can have publically on the steps taken. Let me know if you have any follow up questions on this.

https://github.com/pokt-network/pocket/assets/1892194/c160da23-2854-461f-888f-f162df076dda

**Comment by Olshansk** [comment]
> @adshmh I tried to review the changes since my last review and I'm guessing you `git fooed` to clean up some of your local commits.
> 
> It really helps the reviewer to see what changes took place between each review if we don't modify the git history, especially for PRs that are larger and take longer.
> 
> Per our review guidelines (#773), we always do a "squash and merge" when we merge the PR into main, so the history will be clean on main regardless.
> 
> Screenshot 2023-05-23 at 8 52 00 PM Screenshot 2023-05-23 at 8 51 56 PM

@adshmh Seems like everything is green so feel free to merge if you think it's ready!

**Pull request merged**